content/en/platform/corda/1.0/cenm/workflow.md

```diff
@@ -163,7 +163,7 @@ data class WorkflowPluginRequest(val requestId: String,
 ## Example
 
 This sample workflow plugin creates a request file in *basedir* when the Identity Manager received a certificate signing request, user can then approve or reject the request by moving the request files to *approved* or *rejected* folder.
-The certificate signing process will then issue a certificate for the request (require signer configuration), and move the request files to *done* folder
+The certificate signing process will then issue a certificate for the request (require signer configuration), and move the request files to *done* folder.
 
```

Comment:
Though we're not fixing typos right now really, could you fix this one since the PR is open anyway? It's on this page.
![image](https://user-images.githubusercontent.com/69467840/137485193-e3c3c149-f13a-4d1a-883b-54e624fd75b2.png)